client/js/heat.js

```diff
@@ -35,28 +37,26 @@ function makeBinaryArr(arr, total) {
 /**
  * turns array of data into heat map
  * @param {number[]} data array of all 0s/1s heat map data
- * @param {number} dataLength size of data array
  * @returns {HeatMapRow[]} square matrix of HeatMapValues for HeatMap
  */
-function createHeatMapValues(data, dataLength) {
+function createHeatMapValues(data) {
   // heat map should have equal length and width
-  const numRows = Math.ceil(Math.sqrt(dataLength));
+  const numRows = Math.ceil(Math.sqrt(data.length));
 
   const heatMapValues = [];
-  for (let i = 0; i < dataLength; i += numRows) {
+  for (let i = 0; i < data.length; i += numRows) {
     heatMapValues.push(new HeatMapRow(data.slice(i, i + numRows)));
   }
   return heatMapValues.reverse();
 }
 
 /**
  * renders heat map onto html
- * @param {number[]} data array of all 0s/1s heat map data
- * @param {number} dataLength size of data array
+ * @param {number[]} allBinaryData array of all 0s/1s heat map data
```

Comment:
consider renaming to just `binaryData`, `allBinaryData` insinuates that there's some other array with just a segment of the data, which there isn't so we can simplify this name